linear-base.cabal

```
@@ -21,13 +21,26 @@ source-repository head
     type:     git
     location: https://github.com/tweag/linear-base
 
-common warnings
-    ghc-options: -Wall -Wcompat -Wincomplete-record-updates -Wincomplete-uni-patterns -Wnoncanonical-monad-instances
+common build-opts
+    ghc-options: -O2 -Wall -Wcompat -Wincomplete-record-updates -Wincomplete-uni-patterns -Wnoncanonical-monad-instances
                  -- Additional warnings we may consider adding:
                  -- * -Wredundant-constraints : would need deactivating in the modules which use Nat
+common rts-opts-multithread
+    ghc-options: -threaded -rtsopts -with-rtsopts=-N
+common rts-opts-monothread
+    ghc-options: -threaded -rtsopts -with-rtsopts=-N1
```

Why not simply removing `-threaded`?
```suggestion
    ghc-options: ""
```

---

Apparently there are [some non-negligible](https://discourse.haskell.org/t/make-ghc-threaded-the-default-rts/4220/6) differences, which indicates we should probably use no option at all for the bench case.